### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by eng-eng11**
### Project 1: Bubble Sort
Sorting algorithms are some of the earliest that you typically get exposed to in Computer Science. It may not be immediately obvious how important they are, but it shouldn’t be hard to think of some examples where your computer needs to sort some massive datasets during everyday operations.

One of the simpler (but more processor-intensive) ways of sorting a group of items in an array is bubble sort, where each element is compared to the one next to it and they are swapped if the one on the left is larger than the one on the right. This continues until the array is eventually sorted.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,69 @@
...
+
+2. Build another method `#bubble_sort_by` which sorts an array by accepting a block
+
+``` rb
+# frozen_string_literal: true
+
+# rubocop:disable Style/WordArray
+def bubble_sort_by(array)
+  not_sorted = true, n = array.length - 1
+  while not_sorted
```
Comment (samgaco) [inline README.md:45]: This method is included in the readme file but not in the .rb file

File: bubble_sort.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    (0...index).each do |i|
+      next unless array[i] > array[i + 1]
+
+      array[i], array[i + 1] = array[i + 1], array[i]
+      not_sorted = true
+    end
+    index -= 1
+  end
+end
+
```
Comment (samgaco) [inline bubble_sort.rb:21]: **bubble_sort_by** is missing in this file

**Review by samgaco — CHANGES_REQUESTED** [review]
Hi @falguniislam 

Nice description on the pull request and cool readme file :+1:  I left some comments below 

- (the second required method is not showing in the .rb file).

- This project is collaborative, any reason on why you are not working with your coding partner for this project? If you are experiencing any issues you should get in contact with the student success team.

----------------------


* ###### STATUS OF THE PROJECT: REQUESTED CHANGES :stop_sign:

File: bubble_sort.rb (hunk 2 above)
Comment (eng-eng11) [reply bubble_sort.rb:21]: my partner wants to work alone , thats why i am also working alone. my partner dont' wants to work with me.

**Pull request merged**